news_events/etl/ol_events.py

```diff
@@ -5,171 +5,192 @@
 from urllib.parse import urljoin
 from zoneinfo import ZoneInfo
 
-from bs4 import BeautifulSoup as Soup
 from dateutil import parser
-from requests import HTTPError
+from django.utils.html import strip_tags
 
-from main.constants import ISOFORMAT
+from main.utils import now_in_utc
 from news_events.constants import FeedType
-from news_events.etl.utils import get_soup, safe_html, tag_text
+from news_events.etl.utils import get_request_json
 
 log = logging.getLogger(__name__)
-
 OL_EVENTS_BASE_URL = "https://openlearning.mit.edu/"
+OL_EVENTS_SOURCE_URL = urljoin(OL_EVENTS_BASE_URL, "/events/")
+OL_EVENTS_TITLE = "Open Learning Events"
+# Copied from OL_EVENTS_BASE_URL html page
+OL_EVENTS_DESCRIPTION = """
+Open Learning hosts a wide range of events for learners, educators, researchers,
+nd practitioners around the world.
+Regular event series include Open Learning Talks, which bring together leaders
+to discuss research-based ideas, technologies, and efforts in education; and xTalks,
+which provide MIT faculty, researchers, staff, and students an opportunity to share
+their experiences developing and using digital technologies in the classroom.
+Explore upcoming events below, including webinars, workshops, and more.
+"""
 
 # One URL for now, might switch to a list of audience/type-specific URLs later
-OL_EVENTS_URLS = ["/events"]
+OL_EVENTS_API_URL = (
```

Comment:
should this url build off of OL_EVENTS_BASE_URL?

Reply:
not a blocker since I'm guessing this url may never change but I was wondering if it would be better to have this sit in settings.py with all the other urls

Reply:
The other news_events ETL urls are also stored in their respective python files.  For learning resources, we originally had the urls come from settings because RC and prod would often hit different endpoints for those (RC MITx Online vs prod MITx Online, for example) but recently we moved away from that, hitting the prod versions of those api endpoints all the time.  

If moving to settings, might be best to do all of them at once in a separate PR.